### PR TITLE
Release 2.2.0

### PR DESCRIPTION
**Pull request by anita-steiner**
* Add `long_description_content_type='text/markdown'` to _setup.py_
* Change phovea_server version in _requirements.txt_

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,12 +56,12 @@
     "iojs": ">= 3"
   },
   "dependencies": {
-    "phovea_core": "~2.1.2"
+    "phovea_core": "github:phovea/phovea_core#develop"
```
Comment (thinkh) [inline package.json]: When merging into `master` there should be no dependency to a `develop` branch!

File: phovea_security_flask/flask_login_impl.py [hunk 2]
```diff
@@ -39,6 +39,7 @@ def logout(self, user):
 
 class NamespaceLoginManager(security.SecurityManager):
   def __init__(self):
+    print('init flask login')
```
Comment (thinkh) [inline phovea_security_flask/flask_login_impl.py]: Please use `_log.info(...)`

File: phovea_security_flask/flask_login_impl.py [hunk 3]
```diff
@@ -62,9 +63,11 @@ def _load_user(self, id):
   #  return User.query.get(int(id))
 
   def init_app(self, app):
+    print('init flask app')
```
Comment (thinkh) [inline phovea_security_flask/flask_login_impl.py]: Please use `_log.info(...)`

File: phovea_security_flask/flask_login_impl.py [hunk 4]
```diff
@@ -62,9 +63,11 @@ def _load_user(self, id):
...
     self._manager.init_app(app)
 
   def add_login_routes(self, app):
+    print('add flask routes')
```
Comment (thinkh) [inline phovea_security_flask/flask_login_impl.py]: Please use `_log.info(...)`

File: phovea_security_flask/flask_login_impl.py [hunk 5]
```diff
@@ -109,20 +112,24 @@ def loggedinas():
       return '"not_yet_logged_in"'
 
   def login_required(self, f):
+    print('login_required flask')
```
Comment (thinkh) [inline phovea_security_flask/flask_login_impl.py]: Please use `_log.info(...)`

File: phovea_security_flask/flask_login_impl.py [hunk 6]
```diff
@@ -109,20 +112,24 @@ def loggedinas():
...
     return flask_login.login_required(f)
 
   @property
   def current_user(self):
+    print('current user flask')
```
Comment (thinkh) [inline phovea_security_flask/flask_login_impl.py]: Please use `_log.info(...)`

File: phovea_security_flask/flask_login_impl.py [hunk 7]
```diff
@@ -109,20 +112,24 @@ def loggedinas():
...
     return flask_login.current_user
 
   def logout(self):
+    print('logout flask')
```
Comment (thinkh) [inline phovea_security_flask/flask_login_impl.py]: Please use `_log.info(...)`

File: phovea_security_flask/flask_login_impl.py [hunk 8]
```diff
@@ -109,20 +112,24 @@ def loggedinas():
...
     u = self.current_user
     _log.debug('user logout: ' + (u.name if hasattr(u, 'name') else str(u)))
     for store in self._user_stores:
       store.logout(u)
     flask_login.logout_user()
 
   def login(self, username, extra_fields=None):
+    print('login flask')
```
Comment (thinkh) [inline phovea_security_flask/flask_login_impl.py]: Please use `_log.info(...)`

File: phovea_security_flask/flask_login_impl.py [hunk 9]
```diff
@@ -144,7 +151,7 @@ def _load_user_from_key(self, api_key):
 
   def _load_user_from_request(self, request):
     # first, try to login using the api_key url arg
-    api_key = request.args.get('api_key')
+    api_key = request.headers.get('api_key')
```
Comment (thinkh) [inline phovea_security_flask/flask_login_impl.py]: See my question in https://github.com/phovea/phovea_security_flask/pull/27#issuecomment-524932109

**Comment by thinkh** [comment]
@anita-steiner Do you still need to add the new `publish` task in the _package.json_?

File: package.json (hunk 1 above)
Comment (anita-steiner) [reply package.json]: jab sorry

File: phovea_security_flask/flask_login_impl.py (hunk 2 above)
Comment (anita-steiner) [reply phovea_security_flask/flask_login_impl.py]: I will remove that - was only for an internal test

**Pull request merged**